fre/app/freapp.py

```diff
@@ -7,70 +7,118 @@
 
 from .mask_atmos_plevel import mask_atmos_plevel_subtool
 from .generate_time_averages.generate_time_averages import generate
+from .regrid_xy.regrid_xy import _regrid_xy
 
 @click.group(help=click.style(" - access fre app subcommands", fg=(250,154,90)))
 def app_cli():
     ''' entry point to fre app click commands '''
 
+@app_cli.command()
+@click.option("-i", "--input_dir",
+              type = str,
+              help = "`inputDir` / `input_dir` (env var) specifies input directory to regrid, " + \
+                     "typically an untarredv history file archive" ,
+              required = True)
+@click.option("-o", "--output_dir",
+              type = str,
+              help = "`outputDir` / `output_dir` (env var) specifies target location for output" + \
+                     " regridded files",
+              required = True)
+@click.option("-b", "--begin",
+              type = str,
+              help = "`begin` / `begin` (env var) ISO8601 datetime format specification for" + \
+                     " starting date of data, part of input target file name",
+              required = True)
+@click.option("-t", "--tmp_dir",
+              type = str,
+              help = "`TMPDIR` / `tmp_dir` (env var) temp directory for location of file " + \
+                     "read/writes",
+              required = True)
+@click.option("-r", "--remap_dir",
```

Comment:
Similar comment to above, `-r` usually refers to recursive, consider even `-rd` for remap_dir? (may not be the best still)

Reply:
change implemented next commit